src/Microsoft.AspNetCore.Authentication.AzureADB2C.UI/AzureAdB2CPartFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
Add license headers to this file and everywhere else needed.

Reply:
Yes. I was unblocking the build